sunpy/data/_sample.py

```diff
@@ -14,6 +14,8 @@
 from sunpy.util.net import url_exists
 from sunpy import config
 
+import socket
```

Comment:
Since `socket` is a python library it should be placed in the first block, where `os`, `shutil`, etc...